go.mod

```diff
@@ -30,3 +30,5 @@ require (
 replace github.com/ThalesIgnite/crypto11 => github.com/foundriesio/crypto11 v0.0.0-20221104185643-b2344c63166b
 
 replace github.com/pelletier/go-toml => github.com/foundriesio/go-toml v1.8.1-0.20200721033514-2232fec316b9
+
+replace github.com/foundriesio/go-ecies => ../go-ecies
```

Comment:
don't forget to remove this once you've updated the other package

Reply:
Sure... The github action tests will not pass as long as this line is here.
I actually intentionally did not reference the _exact branch where this works_ in order to break the github action tests.